tests/istio.mk

```diff
@@ -86,6 +86,5 @@ e2e_upgrade: istioctl generate_yaml
 
 e2e_all: e2e_simple e2e_mixer e2e_bookinfo
 
-e2e_pilot:
-	go install ./pilot/test/integration
-	${ISTIO_BIN}/integration --logtostderr ${TESTOPTS} -hub ${HUB} -tag ${TAG}
+e2e_pilot: istioctl generate_yaml
+	go test -v -timeout 20m ./tests/e2e/tests/pilot -hub ${HUB} -tag ${TAG}
```

Comment:
Both prow (in prow/istio-pilot-e2e.sh) and circleCI (in .circleci/config.yml) run e2e_pilot with a bunch of parameters passed in via TESTOPTS (which is now removed) so more work might be needed here (or elsewhere).

Reply:
Good catch @mattdelco ... created #3459 to fix

Reply:
So how do I run this now with parameters?

Reply:
@andraxylia you just add the parameters to the "program arguments" in GoLang (or whatever IDE you're using) ... or from the command line with `go test <dir> <args>`